docs/MIRRORS.md

```diff
@@ -0,0 +1,68 @@
+# Creating & Using Mirrors for Stanza Releases
+
+## Creating a Mirror
+Mirrors for Stanza can come in two forms:  
+1. Hosted websites
+2. Local filesystem mirrors
+
+The only requirements for either are creating a directory layout that  
+mirrors that of the GitHub releases, ie:  
+Example of web URL for latest version: `http://dl.example.com/some/path/latest/download/`  
+Example of web URL for version 1.2.13: `http://dl.example.com/some/path/download/v1.2.13/`  
+Example of web URL for version 1.2.12: `http://dl.example.com/some/path/download/v1.2.12/`  
+Example of file URL for latest version: `file:///Users/username/Downloads/stanza_local/latest/download/`  
+Example of file URL for version 1.2.13: `file:///Users/username/Downloads/stanza_local/download/v1.2.13`
+Example of file URL for version 1.2.12: `file:///Users/username/Downloads/stanza_local/download/v1.2.12`  
+  
+It is suggested to use an automated synchronization process to manage keeping ths up to date, including rewriting the symlink for the latests to the highest version number.
+
+### Mirror Tree Visualization
+➜  stanza_mirror tree
+.
+└── latest
+    └── download
+        ├── stanza-plugins.tar.gz
+        ├── stanza-plugins.zip
+        ├── stanza_darwin_amd64
+        ├── stanza_linux_amd64
+        ├── stanza_linux_arm64
+        ├── stanza_windows_amd64
+        ├── unix-install.sh
+        ├── version.json
+        └── windows-install.ps1
+
+➜  stanza_mirror tree
+.
+└── download
+    └── v1.2.12
+        ├── stanza-plugins.tar.gz
+        ├── stanza-plugins.zip
+        ├── stanza_darwin_amd64
+        ├── stanza_linux_amd64
+        ├── stanza_linux_arm64
+        ├── stanza_windows_amd64
+        ├── unix-install.sh
+        ├── version.json
+        └── windows-install.ps1
```

Comment:
Might be best to combine these as one output, because it can be confusing differentiating where `download` should be in the path. 

```
➜  stanza_mirror tree
.
├── download
│   └── 1.0.0
│       ├── stanza-plugins.tar.gz
│       ├── stanza-plugins.zip
│       ├── stanza_darwin_amd64
│       ├── stanza_linux_amd64
│       ├── stanza_linux_arm64
│       ├── stanza_windows_amd64
│       ├── unix-install.sh
│       ├── version.json
│       └── windows-install.ps1
└── latest
    └── download
        ├── stanza-plugins.tar.gz
        ├── stanza-plugins.zip
        ├── stanza_darwin_amd64
        ├── stanza_linux_amd64
        ├── stanza_linux_arm64
        ├── stanza_windows_amd64
        ├── unix-install.sh
        ├── version.json
        └── windows-install.ps1
```

Also, in your example you do not prefix the version with `v` but in this output you do. Either works as long as the path reflects the version you pass `v1.0.0` | `1.0.0`.

Reply:
We don't have an option when it comes to where download is in the path, that is why I showed both. I see what you're saying though. Smart. I'll edit it tomorrow.

Reply:
Changes made. I've also added examples to the script's built in help. I've also edited the script to automatically prepend the v to the versions in the URLs, as it is required for GitHub URLs. Example from GH: `https://github.com/observIQ/stanza/releases/download/v1.2.13/stanza-plugins.tar.gz` Thus, I've made all URLs require the v to be present in the directory/breadcrumb structure, but not needed to be passed into the script when specifying a version.